### PR TITLE
Remove many static calls to rewrite

### PR DESCRIPTION
**Pull request by ajreynol**
This is the result of a global replace `Rewriter::rewrite(` -> `rewrite(`  + patching the results.

It makes several classes into EnvObj.  Many calls to Rewriter::rewrite remain (that are embedded in classes that should not be EnvObj).

### CODE REVIEW

File: src/theory/fp/fp_word_blaster.h [hunk 1]
```diff
@@ -106,7 +106,7 @@ class nodeWrapper : public Node
  * allowing them to be traced via GDB.
  */
 #ifdef CVC5_SYM_SYMBOLIC_EVAL
-  nodeWrapper(const Node& n) : Node(theory::Rewriter::rewrite(n)) {}
+  nodeWrapper(const Node& n) : Node(theory::rewrite(n)) {}
```
Comment (nafur) [inline src/theory/fp/fp_word_blaster.h:109]: I think this should be different...

**Review by nafur — COMMENTED** [review]
Just one thing...

File: src/theory/fp/fp_word_blaster.h (hunk 1 above)
Comment (ajreynol) [reply src/theory/fp/fp_word_blaster.h:109]: Hm, good catch, apparently this code is disabled?

**Review by nafur — APPROVED** [review]
LGTM

**Pull request merged**